docs/DeveloperGuide.md

```diff
@@ -443,6 +443,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     
           Use case ends.
 
+**Use case: Check eligibility for assistance**
+
+**MSS:**
+
+1. User chooses which person to check eligibility for assistance
+2. SocialBook displays what schemes the person is eligible for
+
+    Use case ends.
+
+**Extensions:**
+
+* 1a. Given index is invalid.
+    * 1a1. SocialBook informs user that the index is invalid
```

Comment:
I think can add the lines below to standardise with the way other use cases resume 
`1a2. User corrects the input and enters the command again.`
`Steps 1a1-1a2 are repeated until the user enters a correct input.`